lib/ansible/config/base.yml

```diff
@@ -2115,4 +2115,13 @@ VERBOSE_TO_STDERR:
     - section: defaults
       key: verbose_to_stderr
   type: bool
+LOCAL_SEARCH_PATH:
+  version_added: '2.17'
+  name: Additional prefered search paths for local files
+  default: '{{ "" }}'
```

Comment:
not sure what you are trying to do here, but his is not what i would accept as a default, remove and just check 'None' as a sentinel value

Reply:
copy & paste form other `pathspec` sorry... now i set it to `[]`